handlers/handler.go

```diff
@@ -95,6 +96,55 @@ func (env *Env) PostUserHandler(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(reqUser)
 }
 
+// GetUserHandler gets a user returning specified columns
+func (env *Env) GetUserHandler(w http.ResponseWriter, r *http.Request) {
+	userID, err := strconv.ParseInt(r.Header.Get("User-ID"), 10, 64)
+	if err != nil {
+		errMsg := "Invalid user ID"
+		log.Println(errMsg + ": " + err.Error())
+		http.Error(w, errMsg, http.StatusBadRequest)
+		return
+	}
+
+	user, err := env.DB.ReadUser(userID)
+	if user == nil {
+		errMsg := "User not found"
+		log.Println(errMsg + ": " + err.Error())
+		http.Error(w, errMsg, http.StatusNotFound)
+		return
+	}
+	if err != nil {
+		internalServerError(w, err)
+		return
+	}
+
+	responseUser := &models.User{}
+	r.ParseForm()
+	includes := r.Form["includes"]
+	if includes == nil {
+		responseUser = user
+		responseUser.Password = ""
+	} else {
+		for _, column := range includes {
+			if column == "id" {
```

Comment:
This might be more readable as a `switch-case` than an `if-else`. You can see the syntax for it [here](https://tour.golang.org/flowcontrol/10).